src/main/java/ru/yandex/practicum/filmorate/storage/impl/FeedDbStorage.java

```diff
@@ -212,31 +153,81 @@ public void deleteReview(Long reviewId) {
         if (userId != -1) {
             Timestamp timestamp = new Timestamp(System.currentTimeMillis());
             sqlQuery =
-                "insert into feeds(userId, timestamp, eventType, operation, entityId)" +
-                    " values(?,?, ?, ?,?)";
+                    "insert into feeds(userId, timestamp, eventType, operation, entityId)" +
+                            " values(?,?, ?, ?,?)";
             jdbcTemplate.update(sqlQuery
-                , userId
-                , timestamp.getTime()
-                , "REVIEW"
-                , "REMOVE"
-                , userId);
+                    , userId
+                    , timestamp.getTime()
+                    , "REVIEW"
+                    , "REMOVE"
+                    , reviewId);
         }
     }
 
-    public Collection<Feed> feeds(Long id){
-        String sqlQuery ="select timestamp, userId, eventType, operation, eventId, entityId from feeds where userId = ?";
-        List<Feed> feeds = jdbcTemplate.query(sqlQuery, this::mapRowToFeed, id);
-        return feeds;
+    public Collection<Feed> feeds(Long id) {
+        String sqlQuery = "select timestamp, userId, eventType, operation, eventId, entityId from feeds where userId = ?";
+        return jdbcTemplate.query(sqlQuery, this::mapRowToFeed, id);
     }
 
     private Feed mapRowToFeed(ResultSet rs, int rowNum) throws SQLException {
         return Feed.builder()
-            .timestamp( rs.getLong("timestamp"))
-            .userId(rs.getInt("userId"))
-            .eventType(rs.getString("eventType"))
-            .operation(rs.getString("operation"))
-            .eventId(rs.getInt("eventId"))
-            .entityId(rs.getInt("entityId"))
-            .build();
+                .timestamp(rs.getLong("timestamp"))
+                .userId(rs.getInt("userId"))
+                .eventType(rs.getString("eventType"))
+                .operation(rs.getString("operation"))
+                .eventId(rs.getInt("eventId"))
+                .entityId(rs.getInt("entityId"))
+                .build();
+    }
+
+    private void addOrRemoveMarkFromUser(long userId, long filmId, String action) {
+        Timestamp timestamp = new Timestamp(System.currentTimeMillis());
+        String sqlQuery2 = "select user_id from users where user_id  = ?";
```

Comment:
If this query is needed to make sure that user with this id exists, we can use a subselect rather than a separate query. Better option would be to let foreign key constraint to take care of this edgecase.